src/components/pages/blog-post/follow-panel.tsx

```diff
@@ -19,7 +19,7 @@ const FollowPanelContainer = styled.div`
   font-size: 16px;
 
   ${up('md')} {
-    margin-top: 0px;
+    margin-top: 0;
```

Comment:
auto fix by your IDE or by purpose? Change is reasonable and okay but should not happen accidentally

Reply:
When I pushed the code my local linter complained about it so I thought why not and removed the "px"